giddy-banana/migrations/20211230140628-create-article-top-editions.cjs

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('ArticleTopEditions', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      TopEditionsId: {
+        type: Sequelize.INTEGER
+      },
+      ArticleId: {
+        type: Sequelize.INTEGER
+      },
+      rank_position: {
+        type: Sequelize.INTEGER
+      },
+      number_of_editions: {
+        type: Sequelize.INTEGER
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
```

Comment:
I am pretty sure we don't have to specify these since sequelize adds them on its own

Reply:
That's right, I didn't have to create it - sequelize did it automatically

Reply:
Yeah, my bad there :(